wscript_build.py

```diff
@@ -335,6 +335,7 @@ def swift(task):
         ( "misc/natural_sort.c" ),
         ( "misc/node.c" ),
         ( "misc/rendezvous.c" ),
+        ( "misc/random.c" ),
```

Comment:
Same for meson?

Reply:
Yes, there's a similar change in meson.build.

Reply:
Somehow I missed this, sorry for the noise.